### DISCUSSION TITLE
Documenting accessibility best practices

### DISCUSSION THREAD

**Discussion by mayank99** [iTwin/iTwinUI-react#226] [Ideas]
Would be nice to document accessibility best practices once we have our own website.

Examples:
https://material-ui.com/components/breadcrumbs/#accessibility
https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb

**Comment by bentleyvk** [comment]
Maybe not only the accessibility but also where to use our components (Do and Don't).
